src/MockBot.Api/Interfaces/IChatService.cs

```diff
@@ -8,8 +8,12 @@ public interface IChatService
 
         public IEnumerable<Chat> FindAll();
```

Comment:
Ideally interfaces should have full XML summary comments. Applies to all interfaces in the PR https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/